docker-compose-tests.yaml

```diff
@@ -0,0 +1,33 @@
+version: '3.6'
+services:
+  postgres:
+    image: postgres:11.2-alpine
+    restart: always
+    volumes:
+    - db_data:/var/lib/postgresql-test/data
+  graphql-engine:
+    image: hasura/graphql-engine:v1.0.0-beta.8
+    ports:
+    - 9000:8080
+    depends_on:
+    - postgres
+    restart: always
+    environment:
+      HASURA_GRAPHQL_DATABASE_URL: postgres://postgres:@postgres:5432/postgres
+      HASURA_GRAPHQL_ENABLE_CONSOLE: "true"
+      HASURA_GRAPHQL_ENABLED_LOG_TYPES: startup, http-log, webhook-log, websocket-log, query-log
+  hasura-service-tests:
+    build:
+      context: ./database-service
+      dockerfile: ./Dockerfile
+      network: host
+      target: unittest
+    restart: "no" 
+    depends_on:
+      - graphql-engine
+    volumes:
+      - ./database-service:/app
+    entrypoint: pytest --hasura-endpoint http://graphql-engine:8080 -ra --junitxml=test-report.xml
```

Comment:
That is how the unit tests are run in the python docker container.

Reply:
I guess it's a matter of taste. I inspired myself of [this example project](https://github.com/dotnet-architecture/eShopOnContainers) where they define their [docker-compose like this](https://github.com/dotnet-architecture/eShopOnContainers/blob/dev/docker-compose-tests.override.yml) and their [Dockerfile this way](https://github.com/dotnet-architecture/eShopOnContainers/blob/dev/src/Services/Basket/Basket.API/Dockerfile). I find it better to have the entrypoint in the docker-compose because I can see what command is run for each test in a single file. There might be advantages to the entrypoint in the `Dockerfile` too. Currently, I'd go on like that and if we see that this is really a huge problem in further developments, then we can always put those entrypoints back to the dockerfiles. 

For sure, however, the entrypoint of the application's image will be in the `Dockerfile`.

Reply:
OK, let us keep it that way